src/components/core/menu/mainMenu/MainMenuRightChainSelector.tsx

```diff
@@ -17,7 +17,7 @@ interface Props {
 
 export const MainMenuRightChainSelector: FC<Props> = ({ networks }) => {
   const activeNetwork = networks.find((network) => network.isCurrentNetwork);
-  if (!activeNetwork) return;
+  if (!activeNetwork || networks.length === 1) return;
```

Comment:
safer to check networks.length < 2

Reply:
activeNetwork would be false if network.length === 0, but I'll change. Either works for me